### ISSUE TITLE
Data error: Number of GWAS participants from the UK

### ISSUE THREAD

**Issue by saloni-nd** [OxfordDemSci/gwasdiversitymonitor#90]
There seems to be an error in the number of GWAS participants on the [dashboard](https://gwasdiversitymonitor.com/).

In 2023, it lists 80 million UK participants, which is more than the total population of the country:

<img width="500" alt="image" src="https://github.com/OxfordDemSci/gwasdiversitymonitor/assets/31684905/a45a5dea-4561-4786-b5eb-a77c0d9f544b">

In 2021, it lists 3 billion UK participants:

<img width="500" alt="image (1)" src="https://github.com/OxfordDemSci/gwasdiversitymonitor/assets/31684905/8eb012bb-f0ca-4173-a2ab-cc765503db9a">

**Comment by crahal** [comment]
This is not an error; people contribute their data to one dataset (e.g. UK Biobank), and then that dataset gets used multiple times...

**Comment by saloni-nd** [comment]
Hi @crahal, thanks for clarifying that. 

It wasn't clear from the dashboard what the number of participants means (i.e. the same dataset reused). However, the 3 billion participants in 2021 is also a few orders of magnitude higher than the years before or after.

**Comment by crahal** [comment]
Yes, sincere apologies; closed this too quickly by mistake without thinking thoroughly enough! I've re-opened it because it is an issue in language, even if not an error in data. Labelling this as an #enhancement and will push a fix very soon! Thank you for flagging this!
